inside.go

```diff
@@ -223,7 +228,7 @@ func (f *Interface) sendNoMetrics(t NebulaMessageType, st NebulaMessageSubType,
 	var err error
 	//TODO: enable if we do more than 1 tun queue
```

Comment:
Is this TODO needing to be auctioned now with multi queue?

Reply:
@forfuncsake No because we use sync/atomic for the messageCounter now. We should remove that outdated TODO.